actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -189,7 +197,9 @@ private void courseProgressMetrics(Request actorMessage) {
     }
 
     if (StringUtils.isBlank(batchId)) {
-      ProjectLogger.log("CourseMetricsActor-courseProgressMetrics-- batch is not valid .");
+      ProjectLogger.log(
+          "CourseMetricsActor:courseProgressMetrics CourseMetricsActor-courseProgressMetrics-- batch is not valid .",
```

Comment:
From:
"CourseMetricsActor:courseProgressMetrics CourseMetricsActor-courseProgressMetrics-- batch is not valid ."

To:
"CourseMetricsActor:courseProgressMetrics: batchId is invalid (blank)."